tests/hash/conftest.py

```diff
@@ -0,0 +1,19 @@
+"""
+    Pytest configuration used by the read generic hash tests.
+"""
+
+
+def pytest_addoption(parser):
+    parser.addoption("--hash_field", action="store", default="random",
+                     help="The hash field to test, can be 'all', 'random' or a designated one")
```

Comment:
is it possible to list all the available ones?

Reply:
Done